webpack/assets/javascripts/react_app/components/common/DateTimePicker/DateTimePicker.stories.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
+import { storiesOf } from '@theforeman/stories';
 import DateTimePicker from './DateTimePicker';
 import Story from '../../../../../../stories/components/Story';
 
-storiesOf('Components/DateTimePicker', module).add('DateTimePicker', () => (
+storiesOf('Components|DateTimePicker', module).add('DateTimePicker', () => (
```

Comment:
not as expected:
![FireShot Capture 200 - Components - DateTimePicker - DateTimePicker ⋅ Storybook - localhost](https://user-images.githubusercontent.com/11807069/71783513-5ef51d00-2ff0-11ea-981e-eef192804c24.png)

Reply:
Fixed and converted to CSF